taxonomy-editor-frontend/src/utils.ts

```diff
@@ -1,17 +1,77 @@
+import { API_URL } from "./constants.js";
+
 /**
  * @returns snake cased string converted into title case. Example: 'hello_friend' to 'Hello Friend'
  */
-export function toTitleCase(taxonomyName: string) {
+export const toTitleCase = (taxonomyName: string) => {
```

Comment:
If you're changing the definition conventions, please do make sure it is followed everywhere.

Reply:
yep, all are following `const`